scene/resources/visual_shader.cpp

```diff
@@ -949,9 +949,6 @@ Vector2 VisualShader::get_node_position(Type p_type, int p_id) const {
 Ref<VisualShaderNode> VisualShader::get_node(Type p_type, int p_id) const {
 	ERR_FAIL_INDEX_V(p_type, TYPE_MAX, Ref<VisualShaderNode>());
 	const Graph *g = &graph[p_type];
-	if (!g->nodes.has(p_id)) {
-		return Ref<VisualShaderNode>();
-	}
 	ERR_FAIL_COND_V(!g->nodes.has(p_id), Ref<VisualShaderNode>());
```

Comment:
This should have a message now, if we don't just remove the error instead, it doesn't tell users anything really

Reply:
That means there is something wrong with the error macros or that is an intended behavior?

Reply:
No they work fine, it's just that people won't really know how to interpret `!g->nodes.has(p_id)`, if they don't know the source code, so something more descriptive would help, like "Graph does not have a node with id `%i`"

Reply:
Okay got it, I will make the change asap. Thanks.